src/glue/js/i64.js

```diff
@@ -27,6 +34,18 @@ global.i64_high = function i64_high(value) {
   return value.high;
 };
 
+global.i64_not = function i64_not(value) {
+  return value.not();
+};
+
+global.i64_clz = function i64_clz(value) {
+  return value.high ? clz32(value.high) : clz32(value.low) + 32;
+};
+
+global.i64_ctz = function i64_ctz(value) {
+  return value.low ? ctz32(value.low) : ctz32(value.high) + 32;
+};
```

Comment:
Seems that we could add these to long.js and reuse here :)

Reply:
That's will be great!